app/src/main/java/org/simple/clinic/patient/fuzzy/PercentageFuzzer.kt

```diff
@@ -14,11 +18,21 @@ class PercentageFuzzer(private val fuzziness: Float) : AgeFuzzer {
     }
   }
 
+  private val oneYearAsDays = 365.25
+
   override fun bounded(age: Int): BoundedAge {
-    if (age < 0) {
-      throw AssertionError("Age cannot be negative")
+    if (age <= 0) {
+      throw AssertionError("Age cannot be negative or zero")
     }
+    val dateToFuzz = LocalDate.now(clock).minusYears(age.toLong())
```

Comment:
I explicitly did not use `dateOfBirth` here, because that implies that `PercentageFuzzer` knows how that date is going to be used. As it is now, `PercentageFuzzer` can be used to fuzz any date, not just a "date of birth".

Reply:
Ah okay, makes sense.